service/src/tests/commons.rs

```diff
@@ -118,7 +118,9 @@ pub fn setup<E: EnclaveBase>(
 ) -> (Api<sr25519::Pair, WsRpcClient>, Option<u32>, ShardIdentifier) {
 	let node_url = format!("ws://{}:{}", "127.0.0.1", port);
 	let mut api = Api::<sr25519::Pair, WsRpcClient>::new(WsRpcClient::new(&node_url)).unwrap();
-	ensure_account_has_funds(&mut api, &enclave_account(enclave_api));
+	if let Err(x) = ensure_account_has_funds(&mut api, &enclave_account(enclave_api)) {
+		error!("Couldn't ensure that enclave has funds {:?}", x);
+	}
```

Comment:
We probably want this to panic if it fails, no? Or do we have tests that expect this to fail?

If not, use `expect()`:

```
ensure_account_has_funds(&mut api, &enclave_account(enclave_api)).expect(format!("Couldn't ensure that enclave has funds {:?}", x));
```